Calibration/HcalCalibAlgos/test/HcalIsoTrkAnalyzer.cc

```diff
@@ -396,21 +449,75 @@ void HcalIsoTrkAnalyzer::analyze(edm::Event const& iEvent, edm::EventSetup const
   t_TracksProp = trkCaloDirections.size();
   t_ietaAll->clear(); t_ietaGood->clear();
 #ifdef EDM_ML_DEBUG
-  edm::LogInfo("HcalIsoTrack") << "# of propagated tracks " << t_TracksProp
-			       << " out of " << t_Tracks << " with Trigger "
-			       << ignoreTrigger_;
+  edm::LogVerbatim("HcalIsoTrack") << "# of propagated tracks " << t_TracksProp
+				   << " out of " << t_Tracks << " with Trigger "
+				   << ignoreTrigger_;
 #endif
+
   //Trigger
-  t_trgbits->clear();
-  t_TracksSaved = 0;
+  t_trgbits->clear(); t_hltbits->clear();
```

Comment:
hi @bsunanda - the manipulations here and just below can be replaced by

t_trgbits->assign(trigNames_.size(),false)

Reply:
done